plugin/storage/es/samplingstore/dbmodel/converter.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2024 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dbmodel
+
+import (
+	"github.com/jaegertracing/jaeger/cmd/collector/app/sampling/model"
+)
+
+func FromThroughputs(throughputs []*model.Throughput) []model.Throughput {
```

Comment:
I don't follow - you are converting between identical objects, just from pointers to values. The point was not to use jaeger/model types in the structs defined in dbmodel/model.go, but re-declare similar structure and then transform between that and jaeger/model

Reply:
got it